citi-onboarding/mural-valores
# Feat/animate svg

## What I did

- Animate SVG curve

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout feat/animate-svg`
   1. `git pull`
   1. `yarn install`
   1. `code .`
   1. `yarn start`

1. In your browser go to http://localhost:3000/<route>

1. Check if the application runs
